src/components/HumanRights.vue

```diff
@@ -0,0 +1,106 @@
+<template>
+  <section class="human-rights">
+    <ul class="human-rights__list">
+      <li class="human-rights__item" v-for="(entity, index) in nodeQuery.entities">
+        <img :width="entity.fieldImagen.derivative.width" :height="entity.fieldImagen.derivative.height" class="human-rights__img" :src="entity.fieldImagen.derivative.url" :alt="entity.entityLabel">
```

Comment:
width and height should be divided by 2, because resolution is doubled in order to have retina support.